DataFormats/TotemDigi/interface/TotemTriggerCounters.h

```diff
@@ -0,0 +1,24 @@
+/****************************************************************************
+*
+* This is a part of the TOTEM offline software.
+* Authors: 
+*   Jan Kašpar (jan.kaspar@gmail.com) 
+*    
+****************************************************************************/
+
+#ifndef DataFormats_TotemDigi_TotemTriggerCounters
+#define DataFormats_TotemDigi_TotemTriggerCounters
+
+/**
+ * Trigger counters from LoneG.
+**/
+struct TotemTriggerCounters
+{
+  unsigned char type;
+  unsigned int event_num, bunch_num, src_id;
+  unsigned int orbit_num;
+  unsigned char revision_num;
+  unsigned int run_num, trigger_num, inhibited_triggers_num, input_status_bits;
```

Comment:
Encapsulation is a core principle in C++.
Now that you made this type into a class, I think that set/get should be implemented.

The initialization is needed because by default the compiler we use doesn't initialize simple types.